src/request.js

```diff
@@ -4,10 +4,29 @@
 // See https://developers.cloudflare.com/workers/reference/cloudflare-features/
 require('whatwg-fetch')
 
+// Available `cf` features. Any keys that are not here and set in `cf`
+// will be discarded.
+const cfFeatures = [
+  "scrapeShield",
+  "polish",
+  "minify",
+  "mirage",
+  "scrapeShield",
+  "apps",
+  "resolveOverride",
+  "cacheKey",
+  "cacheTtl",
+  "cacheTtlByStatus"
+]
```

Comment:
can you also add: `hostMetadata` its something which gets added if your zone has ssl for sass... its not publicly documented anywhere but its something cloudflare shares when they enable ssl for sass for your zone.